src/cluster.ts

```diff
@@ -80,6 +90,103 @@ export type GetClusterMetadataCallback = (
   apiResponse?: IOperation | null
 ) => void;
 
+export type BackupTimestamp = google.protobuf.ITimestamp | PreciseDate | Date;
+export interface ModifiableBackupFields {
+  /**
+   * The ITimestamp (Date or PreciseDate will be converted) representing
+   * when the backup will automatically be deleted. This must be at a
+   * minimum 6 hours from the time of the backup request and a maximum of 30
```

Comment:
this min/max is mentioned but I didn't see anything validating this.

Reply:
from what i found, this min/max was _not_ documented. the grpc response will respond with an error outside of these bounds. i opted to _not_ perform client-side validation in case this changes in the future.